geotrek/common/parsers.py

```diff
@@ -77,6 +77,7 @@ class Parser:
     update_only = False
     delete = False
     duplicate_eid_allowed = False
+    fill_empty_translated_fields = False
```

Comment:
Keep documenting new attributes above

Reply:
Yes, I tried to list all basic settings for parsers, it was not present in documentation.